### DISCUSSION TITLE
How to hide footer?

### DISCUSSION THREAD

**Discussion by micycle1** [datapane/datapane#184] [Help] [answered]
Is there a way to disable/hide the DataPane footer during html export?

![image](https://user-images.githubusercontent.com/9304234/150345367-b16610c1-2b9b-4f3b-a6f3-195afc77e1f1.png)

**Comment by mands** [comment, marked as answer]
Hi there,

Thanks for your question - right now it's not possible to remove the footer during export - we'll investigate other options we can implement here.
